core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -1643,7 +1643,7 @@ public void sendAdventureSettings() {
         boolean spectator = gameMode == GameMode.SPECTATOR;
         boolean worldImmutable = gameMode == GameMode.ADVENTURE || spectator;
 
-        if (org.geysermc.geyser.network.MinecraftProtocol.supports1_19_10(this)) {
+        if (org.geysermc.geyser.network.GameProtocol.supports1_19_10(this)) {
```

Comment:
This no longer needs to be a fully qualified import.

Reply:
https://github.com/GeyserMC/Geyser/pull/3131/commits/2dbd39c5a437ee14c589acb1ab2576c9779725b9